packages/web-components/fast-components/src/card/index.ts

```diff
@@ -8,11 +8,7 @@ import {
 } from "@microsoft/fast-foundation";
 import { CardStyles as styles } from "./card.styles";
 import { parseColorHexRGB } from "@microsoft/fast-colors";
-import {
-    createColorPalette,
-    DesignSystem,
-    neutralFillCard,
-} from "@microsoft/fast-components-styles-msft";
+import { createColorPalette, FASTDesignSystem, neutralFillCard } from "..";
```

Comment:
do you see circular-dependency warnings when you run `yarn build` in fast-components?